src/utils/query.js

```diff
@@ -15,6 +15,13 @@
  */
 import liskAPIInstance from './api';
 
+const handleErrorResponse = (response) => {
+	if (!response.success) {
+		return Promise.reject(new Error(response.message));
```

Comment:
Does this definitely work with all possible responses? With 1.0.0 the Core API will return an array of errors in some cases.

Reply:
We need to change this according to API response, but this is specific to the Core API that `getBlock` etc are using